src/components/Footer/Footer.js

```diff
@@ -0,0 +1,115 @@
+import {
```

Comment:
I have checked your branch locally.
Did you commit your code using `--no-verify flag`?
I see errors related to formatting in both files Footer.js and theme.js

![image](https://user-images.githubusercontent.com/61732059/195558210-452502f7-15b6-4345-bc94-345b9155e2df.png)